### ISSUE TITLE
Swagger-Editor 3.5.7, Responses Examples are not interpreted 

### ISSUE THREAD

**Issue by younss** [swagger-api/swagger-editor#1785]
When you copy past an OpenAPI definition with response examples inside the Swagger editor
Example : https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v3.0/api-with-examples.yaml

The examples are not interpreted and shown in the UI.
```
responses:
        '200':
          description: |-
            200 response
          content:
            application/json:
            examples: 
              foo:
```
It works when you move examples to the Response code level
 
```
responses:
        '200':
          description: |-
            200 response
          examples: 
             foo:
```
Can you Advice please

**Comment by shockey** [comment]
This is an artifact from Swagger 2.0 having examples at the code level - you should not rely on it in OpenAPI 3.0.

**Comment by younss** [comment]
@shockey Thanks of the quick answer.
Of course I don't but do you confirm that there is a bug for supporting OpenAPI example inside Swagger Editor?
Regards

**Comment by shockey** [comment]
@younss, yes, this is an unimplemented feature in Swagger UI: see https://github.com/swagger-api/swagger-ui/issues/3437 for updates about it 😄
